### PR TITLE
save verification to db

### PR DESCRIPTION
**Pull request by darmhoo**
#### What does this PR do?
- (replace me) Explain WHY this PR is required. E.g.: What feature it adds, what problem it solves, …

#### Description of Task proposed in this pull request?
- (replace me) A description of WHAT changed in the codebase. Additionally add the reasoning if it's complex or abstract. 

#### How should this be manually tested (Quality Assurance)?
- Explain how to manually test this feature. For example if changes were made in
the UI or in the API, explain where and if any specific access is needed.

#### What are the relevant pivotal tracker stories?
- [Story ID:  replace text with story id](paste Pivotal Tracker link here)

#### Any background context you want to add (Operations Impact)?
- (replace me) List the steps that must be taken for this PR to work.
E.g.: `npm install`, `Add "github_key" to environment variables, ...`

Be sure to include all systems that needs to be changed or which system is
affected by the change (Ex: Requires Styled Components to be installed and
configured in `babel.rc`, file to be updated).

#### What I have learned working on this feature: [If you don't put anything here, you are doing it wrong!]
- (replace me) I learned how to create a pull request template

### CODE REVIEW

File: app/Http/Controllers/AmortizationController.php [hunk 1]
```diff
@@ -106,7 +106,7 @@ public function recommend(AmmortizationService $service)
             $recommendation = Recommendation::create([
                 "staff_id" => $user->id,
                 "customer_id" => request("customer_id"),
-                "type" => request('type'),
+                "type" => 'recommendation',
```
Comment (teebarg) [inline app/Http/Controllers/AmortizationController.php:109]: Why are we hard coding this?

File: database/migrations/new/2023_02_14_085525_update_recommendation_table.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    public function __construct()
+    {
+        DB::getDoctrineSchemaManager()->getDatabasePlatform()->registerDoctrineTypeMapping('enum', 'string');
+    }
+    public function up()
+    {
+        Schema::table('recommendations', function (Blueprint $table) {
+            //
+            $table->string('type')->change();
```
Comment (teebarg) [inline database/migrations/new/2023_02_14_085525_update_recommendation_table.php:24]: What is it before?

File: database/migrations/new/2023_02_14_085525_update_recommendation_table.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('recommendations', function (Blueprint $table) {
+            //
```
Comment (teebarg) [inline database/migrations/new/2023_02_14_085525_update_recommendation_table.php:36]: Let's add the revert code here

File: routes/api.php [hunk 4]
```diff
@@ -64,7 +64,7 @@
     Route::get('/feedbacks/export', 'FeedbackController@export');
     Route::post('/recommendation', 'AmortizationController@recommend');
     Route::get('/customer-recommendation/{customer}', 'RecommendationController@getRecommendationByCustomer');
-
+    Route::post('saveVerification', 'VerificationController@saveVerificationData');
```
Comment (teebarg) [inline routes/api.php:67]: Can we make use of the existing recommendation post route?

File: app/Http/Controllers/AmortizationController.php (hunk 1 above)
Comment (darmhoo) [reply app/Http/Controllers/AmortizationController.php:109]: this is solefy for recommendation service. while the other one is for verification

File: database/migrations/new/2023_02_14_085525_update_recommendation_table.php (hunk 2 above)
Comment (darmhoo) [reply database/migrations/new/2023_02_14_085525_update_recommendation_table.php:24]: enum(formal, informal) and we werent using it

File: database/migrations/new/2023_02_14_085525_update_recommendation_table.php (hunk 3 above)
Comment (darmhoo) [reply database/migrations/new/2023_02_14_085525_update_recommendation_table.php:36]: ok

**Review by AltaraNg — APPROVED** [review]

**Pull request merged**